ipfs-pinning-service.yaml

```diff
@@ -45,6 +45,25 @@ response. The old pin object is deleted automatically.
 
 Pin object can be removed via `DELETE /pins/{cid-of-pin-object}`
 
+# Custom metadata
+
+Pinning Services are encouraged to add support for additional features by
+leveraging optional `meta` attributes:
+
+- `PinStatus.meta[progress]` –  % progress of ongoing pinning operation
+
+- `PinStatus.meta[receivers] = ['multiaddr1','multiaddr2']` list of peers to connect to to speed up transfer of pinned data
+
+- `Pin.meta[replication]` – could define how many copies service should keep
```

Comment:
What happens if the server doesn't support this, does it just silently ignore it?

Reply:
I think that would be the best strategy for this (and other params) that particular services don't support.

Reply:
I clarified that `meta` attributes are optional and can be ignored or omitted in c9fe147adca97a3e7234008cd4ed80b184ce736d